scripts/global.vars.sh

```diff
@@ -65,7 +65,7 @@ case "${OCTET[0]}.${OCTET[1]}" in
        DNS_SERVERS='10.132.71.40'
           NW2_NAME='Secondary'
           NW2_VLAN=0
-        NW2_SUBNET="${IPV4_PREFIX%.*}.128.0/17"
```

Comment:
If you change this default, then you must ammend the other subnet/network stanzas to this default or else they will break. I suggest that you keep this default and override it in your SH-COLO stanza.